src/app/static/src/js/components/runWorkflow/runWorkflowProgress.vue

```diff
@@ -83,17 +112,35 @@ import WorkflowReportLogDialog from "./workflowReportLogDialog.vue";
 import {
     WorkflowRunSummary,
     WorkflowRunStatus,
+    RunWorkflowMetadata,
+    WorkflowRunReportStatus,
+    WorkflowSummaryResponse,
+    WorkflowReportWithDependencies
 } from "../../utils/types";
 import { buildFullUrl } from "../../utils/api";
+import {BLink} from "bootstrap-vue/esm/components/link";
+import {BCollapse} from "bootstrap-vue/esm/components/collapse";
+import {VBToggle} from 'bootstrap-vue';
+
+interface ExtendedReports extends WorkflowRunReportStatus, WorkflowReportWithDependencies {}
+
+interface ExtendedWorkflowRunStatus extends WorkflowRunStatus {
+    reports: ExtendedReports[]
+}
```

Comment:
This is not a proper way of applying inheritance. Some children interfaces have the same attributes as their parent and each other. I think you should create a separate interface for this data.